README.md

```diff
@@ -60,12 +60,18 @@ npm i --save-dev enzyme
 Enzyme is currently compatible with `React 15.x`, `React 0.14.x` and `React 0.13.x`. In order to
 achieve this compatibility, some dependencies cannot be explicitly listed in our `package.json`.
 
-If you are using `React 0.14` or `React 15.x`, in addition to `enzyme`, you will have to ensure that
+If you are using `React 0.14` or `React <15.5`, in addition to `enzyme`, you will have to ensure that
 you also have the following npm modules installed if they were not already:
 
 ```bash
-npm i --save-dev react-addons-test-utils
-npm i --save-dev react-dom
+npm i --save-dev react-addons-test-utils react-dom
+```
+
+If you are using `React >=15.5`, in addition to `enzyme`, you will have to ensure that you also have
+the following npm modules installed if they were not already:
+
+```bash
+npm i --save-dev react-dom create-react-class
```

Comment:
There's one subtle difference. If you pair create-react-class with react-dom < 15.5.0 then the callback param of `replaceState` won't work.

Otherwise I've tested back to 0.14.0 and I think it's okay.